### ISSUE TITLE
Suggestion: Self-Deactivation

### ISSUE THREAD

**Issue by Deliphin** [CursedFlames/CubeLoader#1]
I thought of this after reading about [The Weirding Gadget](https://minecraft.curseforge.com/projects/the-weirding-gadget) mod.

Basically, after a configurable duration since the last time the player was online, the cube loader would stop loading. It could be reactivated by anyone right clicking it. Maybe have a configurable option to have it autostart once the player rejoins and its block gets loaded, though personally I would have it require to be manually reactivated.

It would also need to be able to support multiple players connected to the same cube loader so groups of people don't have to worry about it.

**Comment by CursedFlames** [comment]
Yep, this is a planned feature. I haven't touched this mod in a while, but I'll probably start working on it again soon.
